src/acceptance-test/java/net/consensys/orion/acceptance/send/receive/privacyGroup/DualNodesPrivacyGroupsTest.java

```diff
@@ -292,21 +290,48 @@ void createAndDeleteTwice() {
     final String[] addresses = new String[] {PK_1_B_64, PK_2_B_64};
 
     // create a privacy group
-    final PrivacyGroup privacyGroup = createPrivacyGroupTransaction(firstNode, addresses, PK_1_B_64, name, description);
+    final PrivacyGroup privacyGroup = createPrivacyGroup(firstNode, addresses, PK_1_B_64, name, description);
 
     final String privacyGroupId = privacyGroup.getPrivacyGroupId();
     assertEquals(privacyGroup.getName(), name);
     assertEquals(privacyGroup.getDescription(), description);
 
     // find the created privacy group in first node
-    final PrivacyGroup[] initialPrivacyGroups = findPrivacyGroupTransaction(firstNode, addresses);
+    final PrivacyGroup[] initialPrivacyGroups = findPrivacyGroup(firstNode, addresses);
     assertEquals(initialPrivacyGroups.length, 1);
     assertEquals(initialPrivacyGroups[0].getPrivacyGroupId(), privacyGroupId);
 
     // delete the privacy group
-    deletePrivacyGroupTransaction(firstNode, privacyGroupId, PK_1_B_64);
+    deletePrivacyGroup(firstNode, privacyGroupId, PK_1_B_64);
 
     // try to delete the group again
-    assertThrows(AssertionFailedError.class, () -> deletePrivacyGroupTransaction(firstNode, privacyGroupId, PK_1_B_64));
+    assertThrows(AssertionFailedError.class, () -> deletePrivacyGroup(firstNode, privacyGroupId, PK_1_B_64));
+  }
+
+  @Test
+  void LegacyPrivacyGroupAddedToQueryPrivacyGroupIfNonLegacyGroupHasBeenCreatedBefore() {
```

Comment:
Nit: start the method name with lowercase :)

Reply:
Ooops
Done!